ngraph/test/backend/quantize_dequantize.in.cpp

```diff
@@ -158,7 +136,7 @@ NGRAPH_TEST(${BACKEND_NAME}, dequantize_zero_offset)
     auto offset = op::Constant::create(input_type, scale_offset_shape, {0});
     auto dequantize = make_shared<op::Dequantize>(X, scale, offset, output_type, quantization_axes);
     auto f = make_shared<Function>(dequantize, ParameterVector{X});
-
+    #if 0
```

Comment:
Please remove this pre-processing instance, we only need the #else part of this code